### DISCUSSION TITLE
Open/run notebook in VSCode

### DISCUSSION THREAD

**Discussion by aarondel4** [microsoft/azuredatastudio#24991] [Q&A] [answered]
I had a notebook I created in Azure Data Studio, but recent updates in our environment have rendered ADS broken and it seems like a massive time sink trying to troubleshoot the electron errors.

I have a working install of VSCode and can open the notebooks in it instead.  Is there a Kernel/extension that enables queries against MS SQL to be run from a ipynb notebook in VS Code?  If so what is it called in the marketplace?

![image](https://github.com/microsoft/azuredatastudio/assets/129524357/2ab42ab6-f760-4510-9789-1fbbc46b9bef)

**Comment by Charles-Gagnon** [comment, marked as answer]
Sorry to hear you're having problems, could you copy these errors you're seeing so we can look into it?

VS Code has the [Polyglot Notebooks](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode) extension which supports SQL & Kusto connections if you wanted to try that.

**Reply by aarondel4** [reply]
Thanks, I'll try that.  This is the error on launching.  I don't have admin access so troubleshooting I can do without pairing up with a non-dev admin is limited.

![image](https://github.com/microsoft/azuredatastudio/assets/129524357/d20d3583-a946-4551-86df-07faa8c63d56)

**Comment by Charles-Gagnon** [comment]
Thanks! It looks like the ADS install is corrupted somehow (the file it's looking for definitely should exist) - so I'd reach out to whoever has admin access and let them know so they can hopefully reinstall it for you.
